modules/doc/content/getting_started/installation/offline_installation.md

```diff
@@ -39,8 +39,8 @@ If you are operating on a personal Linux workstation, the easiest way to obtain
 via the same tool used to obtain a developers environment in the Prerequisites section.
 
 !alert note title=Personal Air-Gapped Workstation
-Procuring the above on a personal air-gapped workstation will need to be your responsibility. We
-cannot instruct a means for bypassing such security.
+Procuring the above on a machine with no network access is beyond the scope of this document. Please
```

Comment:
```suggestion
Procuring the above on a workstation with no network access is beyond the scope of this document. Please
```

HPC cluster is within right?

Reply:
I am not sure I understand the question.

HPC is listed on another bullet point. I tried to separate them as best I could:

#### Air-Gapped HPC Clusters

and

#### Personal Workstation

The idea was not to make these too large, or use ## to include them in the TOC (top right table of contents). But make them large enough to separate them as sections.

Reply:
I thought about indenting them (like we here for each section: https://mooseframework.inl.gov/help/troubleshooting.html) But we quickly run out of space :(

Sometimes I wish we would do away with the left margin!!!